xrdp/xrdp_mm.c

```diff
@@ -1244,6 +1355,7 @@ dynamic_monitor_initialize(struct xrdp_mm *self)
 int
 xrdp_mm_drdynvc_up(struct xrdp_mm *self)
 {
+    struct display_size_description *ignore_marker;
```

Comment:
Should be `struct dynamic_monitor_description *`

`struct display_size_description` is smaller than `struct dynamic_monitor_description`. Using the smaller structure results in invalid memory accesses in `dynamic_monitor_process_queue()` when the pointer is cast to `struct dynamic_monitor_description *`

Valgrind found this one.

Reply:
Fixed.